### ISSUE TITLE
Add starter tests for FriendService

### ISSUE THREAD

**Issue by JohnStarich** [JohnStarich/ee461l#65]
Add tests for the friends API routes.
Use the UserServiceTest for reference.

**Comment by JohnStarich** [comment]
Hey @jjalfaro9, any new progress on this one?

**Comment by jjalfaro9** [comment]
Not at all. I have a 32-bit machine with an older version of MongoDB, so the tests could not load up an instance of Mongo. sorry guys.
